backend/z_index.js

```diff
@@ -273,6 +276,68 @@ app.post('/meeting/:meetingType', async (req, res) => {
     }
   });
 })
+
+//endpoint called when member checks-in to a meeting (upon clicking submit on the check-in form)
+//name, meeting type, and code sent in from frontend (from the form)
+app.post('/updateCheckin', async (req, res) => {
+  const body = req.body
+  const name= body.name
+  const meetingType = body.type
+  const code= body.code
+  var pageId = "";
+  //check if the meeting type is active at this time
+  if (meetingType == "Engineering") {
+    pageId = process.env.NOTION_ENGINEERING_MEETING_INFO;
+  }
+  else if (meetingType == "General") {
+    pageId = process.env.NOTION_GENERAL_MEETING_INFO;
+  }
+  else if (meetingType == "Product") {
+    pageId = process.env.NOTION_PRODUCT_MEETING_INFO;
+  }
+  else if (meetingType == "Design") {
+    pageId = process.env.NOTION_DESIGN_MEETING_INFO;
+  }
+  const response = await notion.pages.retrieve({ page_id: pageId });
+  var activeMeeting = false;
+  // is an empty array if manually deleted otherwise an empty string
+  if (response.properties['Meeting Code'].title.length && response.properties['Meeting Code'].title[0].plain_text !== '') {
```

Comment:
I think we should check if Date.now() < response.properties['End Time'].number instead to set the active meeting setting.  My bad on this cause I had this logic before in my code.